scanpipe/models.py

```diff
@@ -1793,12 +1793,12 @@ def profile(self, print_results=False):
                 print(output_str)
 
 
-def posix_regex_to_django_regex_lookup(regex_pattern):
+def convert_glob_to_django_regex(glob_pattern):
     """
     Convert a POSIX-style regex pattern to an equivalent pattern compatible with the
     Django regex lookup.
```

Comment:
We may need to update the docstring as well.

Reply:
Updated, thanks!